packages/react-native-reanimated/jest-setup.js

```diff
@@ -1,2 +1,21 @@
 delete global.MessageChannel;
 require('./src/jestUtils').setUpTests();
+
+function formatMessage(msg) {
+  return `[Reanimated] ${msg}`;
+}
+
+global.__reanimatedLogger = {
+  warn(msg) {
+    console.warn(formatMessage(msg));
+  },
+  error(msg) {
+    console.log(formatMessage(msg));
+  },
+  fatal(msg) {
+    console.log(formatMessage(msg));
+  },
+  newError(msg) {
+    return new Error(formatMessage(msg));
+  }
+}
```

Comment:
Shouldn't this go into `mocks`?

Reply:
You mean the `packages/react-native-reanimated/src/mock.ts` file?